elasticsearch-1.5.2/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I don't think we want to include these files in the repo. Maybe it would be good to add `elasticsearch-1.5.2/` and `elasticsearch-1.5.2.zip` to the `.gitignore`.